plugins/zlux-plugins/action.yml

```diff
@@ -87,8 +90,34 @@ runs:
       run: |
         cd ../zlux/zlux-build 
         ant removeSource
+        cd ../$REPOSITORY_NAME/web
+        find . -type f -name '*.map*' -exec rm {} \;
+        if [ -f "main.js.gz" ]; then
```

Comment:
this will only capture 1 gz file.
there can be many more.
there can be .css.gz, foo.js.gz, .html.gz
anything .gz

Instead, do this:

`find . -name "*.gz" -exec sh -c 'name=${1%???}; if [ -f "$name" ]; then; rm "$name"; fi;' - {} \;`

this sets name to the name of the .gz file, minus ".gz", and then if such a file exists, it deletes it.